src/Parser/BloodDeathKnight/Modules/Items/T20_2pc.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import SpellIcon from 'common/SpellIcon';
+import { formatPercentage } from 'common/format';
+
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+class T20_2pc extends Module {
+  static dependencies = {
+    combatants: Combatants,
+  }
+
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasBuff(SPELLS.BLOOD_T20_2PC_GRAVEWARDEN.id);
+    console.log('You have the 2pc');
+  }
+
+  item() {
+    const T20_2PC_Uptime = this.owner.modules.combatants.getBuffUptime(SPELLS.BLOOD_T20_GRAVEWARDEN_BUFF.id);
```

Comment:
Please start variables with lower case characters. This could be called `uptime` since the context of the T20_2pc class makes the prefix unnecessary.

Reply:
fixed